test/API/H5_api_attribute_test.c

```diff
@@ -1306,7 +1306,7 @@ test_create_attribute_invalid_params(void)
         PART_BEGIN(H5Acreate_by_name_invalid_aapl)
         {
             TESTING_2("H5Acreate_by_name with invalid AAPL");
-#ifndef NO_INVALID_PROPERTY_LIST_TESTS
+            //#ifndef NO_INVALID_PROPERTY_LIST_TESTS
```

Comment:
Okay, I can remove the commented out code. I was mostly concerned because I added flag checks in some files (especially in tests with testhdf5 as the header), and I was worried that I might have added the wrong checks, so I wanted to keep an easy way to check and verify.

Reply:
Sure, but hopefully review will catch any flag changes that need to be made. Plus it makes it easier to read the diff here when the code is removed instead of being commented out and it makes sure those commented out lines don't end up getting left behind.

Reply:
Got it, I am removing the commented out code right now.